src/components/RecipeDetails.tsx

```diff
@@ -1,7 +1,62 @@
-import React from 'react';
+import React, { FC } from "react";
+import {
+  IInstructions,
+  IInstructionsIngredients,
+  ISteps,
+  IEquipment,
+} from "../interfaces/Recipe";
+import { v4 as uuidv4 } from 'uuid';
 
-const RecipeDetails = ()=> {
-    return <>recipe card</>
+interface RecipeDetailsProps {
+  recipeInstructions: IInstructions[];
+  selectedRecipeImage: string;
 }
 
-export default RecipeDetails;
\ No newline at end of file
+const RecipeDetails: FC<RecipeDetailsProps> = ({
+  recipeInstructions,
+  selectedRecipeImage,
+}) => {
+  const displayStepsInfo = (steps: ISteps[]) => {
+    return steps.map((stepObject) => {
+      const { number, step, ingredients, equipment, length } = stepObject;
+      return (
+        <div key={uuidv4()}>
+          <h2>{`Step ${number}`}:</h2>
+          <p>{`${step}`}</p>
```

Comment:
It seems this uuid package helped get rid of that massive file containing the ingredients?

Reply:
Yup! There were some `key` errors, probably due to duplicate `ingredients` in this component, so I had to use `uuid` instead of `ingredient.id`

Reply:
It's kind of a band-aid solution when you have no better option.  It's possible, yet extremely unlikely for two `uuid`s to be the same.  Extremely, extremely unlikely, but possible.